dali/operators/decoder/nvjpeg/nvjpeg_decoder_decoupled_api.h

```diff
@@ -241,7 +243,7 @@ class nvJPEGDecoder : public Operator<MixedBackend>, CachedDecoderImpl {
       }
       if (host_memory_padding_jpeg2k > 0) {
         nvjpeg_memory::AddBuffer<mm::memory_kind::pinned>(nvjpeg2k_thread_id,
-                                 host_memory_padding_jpeg2k);
+                                                          host_memory_padding_jpeg2k);
```

Comment:
Can't we make this one regular host memory as well? I'd leave a note if there is a good reason why we didn't change that one

Reply:
Nope, nvjpeg2k crashed with regular host buffer.